playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -101,18 +101,11 @@ EDXAPP_MARKETO_CLIENT_ID: ''
 EDXAPP_MARKETO_CLIENT_SECRET: ''
 EDXAPP_MARKETO_LIST_ID: 0
 EDXAPP_MONGODB_LOG:
```

Comment:
See secure PR for the change there.
Or rather, that we're no longer overriding this.